models/maat/R/model2netcdf.MAAT.R

```diff
@@ -89,6 +89,7 @@ model2netcdf.MAAT <- function(outdir, sitelat=-999, sitelon=-999, start_date=NUL
     output[[1]] <- out.year                       # Simulation year
     output[[2]] <- sub.maat.doy+day.steps         # Fractional day - NEED TO IMPLEMENT  
     output[[3]] <- (sub.maat.output$A)            # assimilation in umolsC/m2/s
+    output[[4]] <- (sub.maat.output$gs)           # stomatal conductance in ???
```

Comment:
Need to confirm units then output gs in CF convention at the bottom of the script